django/db/backends/sqlite3/schema.py

```diff
@@ -236,6 +241,14 @@ def is_self_referential(f):
                 mapping[create_field.column] = self.prepare_default(
                     self.effective_default(create_field),
                 )
+        # Alter field pointing to the model itself.
+        if alter_self_field:
+            old_self_field, new_self_field = alter_self_field
+            body.pop(old_self_field.name, None)
+            mapping.pop(old_self_field.column, None)
+            body[new_self_field.name] = new_self_field
+            mapping[new_self_field.column] = self.quote_name(old_self_field.column)
+            rename_mapping[old_self_field.name] = new_self_field.name
```

Comment:
We're doing the same for `alter_field`. It's probably worth to reuse the same code for both, by changing the `alter_field` argument to `alter_fields` (`List[Tuple]`, it was a list before 767849b765e881ee905f2bc1175d1c1f47445554) or by:
```python
        # Add in any altered fields
        for alter_field in [alter_field, alter_self_field]:
            if alter_field:
                old_field, new_field = alter_field
                ...
```

Reply:
Thanks for the review @felixxm .  ```alter_self_field``` is only to be used for altering a m2m self field. I'm afraid passing both in a single argument would create a problem for altering non-m2m field.

Reply:
I've checked and it works fine.

Reply:
Made changes according to second method 👍

Reply:
I don't see any changes.

Reply:
Haven't pushed yet,was waiting for your suggestions on the comment.

Reply:
As far as I'm aware, changing to `alter_fields` makes this API clearer. I pushed edits, please take a look.

Reply:
Thanks a lot! Looks better now.